tools/assetpipelineeditor/orbitcameracontroller.cpp

```diff
@@ -330,31 +330,31 @@ QSize OrbitCameraController::windowSize() const
 
 void OrbitCameraController::setPanSpeed(float v)
 {
-    if (m_panSpeed != v) {
+    if (!qFuzzyCompare(m_panSpeed, v)) {
```

Comment:
is this behaviour wanted in setters ?

Reply:
yeah, we always compare new value to old one, avoid emitting change signal if value has not actually changed